vochain/start.go

```diff
@@ -23,9 +23,11 @@ import (
 	tmjson "github.com/tendermint/tendermint/libs/json"
 	tmlog "github.com/tendermint/tendermint/libs/log"
 	tmos "github.com/tendermint/tendermint/libs/os"
-	"go.vocdoni.io/dvote/log"
+	logger "go.vocdoni.io/dvote/log"
 )
 
+var log = logger.Named("vochain")
```

Comment:
nit: I would call the imported package `logpkg` rather than `logger`, because then it's more obvious that the lines wanting to print a log line shouldn't use `logger`.